doc/usecases/polyculture/incomplete_groups.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: Incomplete Groups
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user is informed about what plant groups need more planning.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app
+  - The plants layer is visible
+  - At least one plant is placed
+  - At least one of the placed plants is in need of another companion plant.
+- **Main success scenario:**
+  - Groups which need more companions are visually highlighted.
+- **Alternative scenario:**
```

Comment:
If there are no groups, then it would be a nice to have to indicate that all plants are happy. This could be also part of gamification. (@ThorbenStaufer)